portal/views/tou.py

```diff
@@ -66,7 +89,63 @@ def get_tou(user_id):
     if not user:
         abort(404)
     current_user().check_role(permission='view', other_id=user_id)
-    tous = ToU.query.join(Audit).filter(Audit.user_id==user_id).first()
+
+    tous = ToU.query.join(Audit).filter(Audit.user_id == user_id)
+
+    return jsonify(tous=[d.as_json() for d in tous])
+
+
+@tou_api.route('/user/<int:user_id>/tou/<string:tou_type>')
+@oauth.require_oauth()
+def get_tou_by_type(user_id, tou_type):
+    """Access Terms Of Use info for given user & ToU type
+
+    Returns ToU{'accepted': true|false} for requested user given the
```

Comment:
OK, cool. So now if a result is found, it'll return something like this:
```
{
  "accepted": "2017-05-22T17:34:24", 
  "agreement_url": "predates agreement_url", 
  "id": 5, 
  "type": "privacy policy"
}
```
whereas if nothing is found, it'll just return
```
{
  "accepted": false
}
```